src/platforms/dart/configuration/integrations/logging.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Logging Integration
+caseStyle: canonical
+supportLevel: production
+sidebar_order: 3
+---
+
+The `sentry_logging` library provides [Logging](https://pub.dev/packages/logging) support for Sentry via [onRecord property](https://pub.dev/documentation/logging/latest/logging/Logger/onRecord.html) that sends events and breadcrumbs to Sentry. Once this integration is configured you can use Logging’s public API.
+
+The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/logging/lib/src).
```

Comment:
What about pointing to the root of the package so users land on the readme and see the test/src projects?

```suggestion
The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/logging/).
```